app/components/show-header/template.hbs

```diff
@@ -1,19 +1,27 @@
 <div class="static-main-header">
-  <div class="col" id="wqxr-logo" data-test-element="header-logo">{{nypr-a-svg className="site-header-title-logo" icon="wqxr-log-frq"}}</div>
+  <div class="col" id="wqxr-logo" data-test-element="header-logo">
+    {{nypr-a-svg className="site-header-title-logo" icon="wqxr-log-frq"}}</div>
```

Comment:
you can clean this up a bit by moving the `</div>` to a new line at the same indentation level as its corresponding `<div>`